plugins/packages/mssql/lib/manifest.json

```diff
@@ -57,6 +57,12 @@
       "type": "text",
       "description": "Enter host"
     },
+    "instanceName": {
+      "label": "Instance",
+      "key": "instanceName",
```

Comment:
currently all the keys in manifests are in snake case, it'd be better not to mix up